Golem.Tools/GolemRunnable.cs

```diff
@@ -26,7 +26,7 @@ protected GolemRunnable(string dir, ILogger logger)
             _logger = logger;
         }
 
-        public abstract bool Start();
+        public abstract bool Start(string? working_dir_name = null);
```

Comment:
I needed it to run `app.exe` in different work dirs to test multiple activities, so the logs would not get mixed together.

Reply:
Because I needed to start app.exe two times in different working dirs, and I preferred to do it this way instead of creating two separate file structures. Nvm, I deleted it already (not pushed yet).